vignettes/motivation.Rmd

```diff
@@ -0,0 +1,67 @@
+---
+title: "Motivation for S7"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Motivation for S7}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+R already has two OO systems built-in (S3 and S4) and many additional OO systems are available in CRAN packages.
+Why did we decide another system (S7) was necessary?
```

Comment:
This breaks from the narrative that we are building on top of S3. I think we really want to avoid arguing for a new object system. I would frame it around how can we learn from S4 to improve S3.